### PR TITLE
Update client and sample for productionunit endpoints

### PR DESCRIPTION
**Pull request by shriharipathak**
OpenApiSpecification updated for new endpoints and models. Client and sample updated for the same.

### CODE REVIEW

File: src/Kmd.Logic.Cvr.Client/CvrClient.cs [hunk 1]
```diff
@@ -92,6 +92,72 @@ public async Task<IList<CvrProviderConfigurationModel>> GetAllCvrConfigurationsA
...
+                                cvr: cvr,
+                                configurationId: this.options.CvrConfigurationId).ConfigureAwait(false))
+            {
+                switch (response.Response.StatusCode)
+                {
+                    case System.Net.HttpStatusCode.OK:
+                        return (IList<ProductionUnit>)response.Body;
+
+                    case System.Net.HttpStatusCode.NotFound:
+                        return null;
```
Comment (base2browns) [inline src/Kmd.Logic.Cvr.Client/CvrClient.cs:118]: Normally you'd return an empty list in this case? What does NotFound mean?

File: src/Kmd.Logic.Cvr.Client/CvrClient.cs [hunk 2]
```diff
@@ -92,6 +92,72 @@ public async Task<IList<CvrProviderConfigurationModel>> GetAllCvrConfigurationsA
...
+
+                    default:
+                        throw new CvrConfigurationException(response.Body as string ?? "Invalid configuration provided to access CVR service");
+                }
+            }
+
+            throw new NotImplementedException();
```
Comment (base2browns) [inline src/Kmd.Logic.Cvr.Client/CvrClient.cs:125]: hmm?

File: src/Kmd.Logic.Cvr.Client/CvrClient.cs [hunk 3]
```diff
@@ -92,6 +92,72 @@ public async Task<IList<CvrProviderConfigurationModel>> GetAllCvrConfigurationsA
...
```
Comment (base2browns) [inline src/Kmd.Logic.Cvr.Client/CvrClient.cs:158]: hmm?

File: sample/Kmd.Logic.Cvr.Client.Sample/Program.cs [hunk 4]
```diff
@@ -64,7 +64,7 @@ private static async Task Run(AppConfiguration configuration)
                 var configs = await cvrClient.GetAllCvrConfigurationsAsync().ConfigureAwait(false);
                 if (configs == null || configs.Count == 0)
                 {
-                    Log.Error("There are no CVR configurations defined for this subscription");
+                    Log.Information("There are no CVR configurations defined for this subscription");
```
Comment (mlatoszek) [inline sample/Kmd.Logic.Cvr.Client.Sample/Program.cs:67]: Why information while you're going to quit the app without successful result?

File: src/Kmd.Logic.Cvr.Client/CvrClient.cs (hunk 1 above)
Comment (mlatoszek) [reply src/Kmd.Logic.Cvr.Client/CvrClient.cs:118]: Cvr not exists?

File: sample/Kmd.Logic.Cvr.Client.Sample/Program.cs (hunk 4 above)
Comment (shriharipathak) [reply sample/Kmd.Logic.Cvr.Client.Sample/Program.cs:67]: Yes, by program behavior that's right. I changed as I got suggestion to change it to information as we are able to get successful response but user does not have any configs to work with.

File: src/Kmd.Logic.Cvr.Client/CvrClient.cs (hunk 2 above)
Comment (shriharipathak) [reply src/Kmd.Logic.Cvr.Client/CvrClient.cs]: Removed.

File: src/Kmd.Logic.Cvr.Client/CvrClient.cs (hunk 3 above)
Comment (shriharipathak) [reply src/Kmd.Logic.Cvr.Client/CvrClient.cs]: Removed.

**Pull request merged**